Configure MinervaNeue as mobile theme

amrav opened the pull request:
Previously this was bundled with MobileExtension, but now we need to
download and configure it manually.

amrav committed:
Configure MinervaNeue as mobile theme

Previously this was bundled with MobileExtension, but now we need to
download and configure it manually.

amrav committed:
Add basic integration test for mobile

icyflame commented:
@amrav I just set this up on a test droplet and it looks exactly as our original wiki does. (on mobile) Is this a feature that was bundled in before and now has been separated out of the theme? Is this a change that is required because we updated to 1.30?

amrav commented:
Yes I believe so. Added a test to catch the regression.

On Mon, 5 Feb 2018 at 09:27, Siddharth Kannan <notifications@github.com>
wrote:

> @amrav <https://github.com/amrav> I just set this up on a test droplet
> and it looks exactly as our original wiki does. (on mobile) Is this a
> feature that was bundled in before and now has been separated out of the
> theme? Is this a change that is required because we updated to 1.30?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/metakgp/metakgp-wiki/pull/36#issuecomment-363027282>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABIZ_w6oQidGCt7HJKIjcayh7GfNzd6Kks5tRsmNgaJpZM4R1BtX>
> .
>

amrav committed:
Merge branch 'master' into amrav/minerva
